includes/Core/Modules/Modules.php

```diff
@@ -25,6 +25,8 @@
 use Google\Site_Kit\Modules\Site_Verification;
 use Google\Site_Kit\Modules\Tag_Manager;
 use Google\Site_Kit\Modules\Ads;
+use Google\Site_Kit\Core\Util\Feature_Flags;
+use Google\Site_Kit\Modules\Reader_Revenue_Manager;
```

Comment:
While we're at it, can we re-order these alphabetically? It might also be nice to remove the unused `OAuth_Client` symbol.

Reply:
Updated.